test/spec/core/jhipster/field_types_test.js

```diff
@@ -9,23 +9,23 @@ const Validations = require('../../../../lib/core/jhipster/validations').VALIDAT
 const JDLEnum = require('../../../../lib/core/jdl_enum');
 
 describe('FieldTypes', () => {
-  describe('::isSQLType', () => {
+  describe('::isCommonDBType', () => {
```

Comment:
This test category (`::isCommonDBType`) is present twice in the file (one for `SQLTypes` and `MongoDBTypes`). Can you please remove one of them? thanks

Reply:
Exactly, new commit soon!